cmd/sbrmi/src/cpuid.rs

```diff
@@ -0,0 +1,1350 @@
+//
+// The MIT License (MIT)
+//
+// Copyright (c) 2015 Gerd Zellweger
```

Comment:
https://github.com/gz/rust-cpuid/pull/137 is the upstream PR that puts formatting into the upstream crate.